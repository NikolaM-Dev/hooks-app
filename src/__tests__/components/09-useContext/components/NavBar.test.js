import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import NavBar from 'components/09-useContext/components/NavBar';

describe('Tests on <NavBar /> component', () => {
  it('should show <NavBar /> correclty', () => {
    const wrapper = shallow(<NavBar />);
    const navLinks = wrapper.find('NavLink');

    expect(wrapper).toMatchSnapshot();
    expect(navLinks).toHaveLength(3);
  });
});
