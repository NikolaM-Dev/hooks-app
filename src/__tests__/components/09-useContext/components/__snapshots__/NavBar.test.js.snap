// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <NavBar /> component should show <NavBar /> correclty 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark bg-dark"
>
  <div
    className="container-fluid"
  >
    <Link
      className="navbar-brand"
      to="/"
    >
      useContext
    </Link>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      >
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/"
        >
          Home
        </NavLink>
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/about"
        >
          About
        </NavLink>
        <NavLink
          activeClassName="active"
          className="nav-link"
          exact={true}
          to="/login"
        >
          Login
        </NavLink>
      </div>
    </div>
  </div>
</nav>
`;
